api/src/main/java/org/apache/iceberg/HistoryEntry.java

```diff
@@ -29,12 +29,12 @@
  */
 public interface HistoryEntry extends Serializable {
   /**
-   * @return the timestamp in milliseconds of the change
+   * Returns the timestamp in milliseconds of the change.
    */
   long timestampMillis();
 
   /**
-   * @return ID of the new current snapshot
+   * Returns iD of the new current snapshot.
```

Comment:
Was this meant to be `id` or `ID`?

Reply:
Ah sorry, I used auto fix for most of this so it probably imagined it was fixing the style. I'll fix that